solutions/swb-app/src/projectRoutes.ts

```diff
@@ -4,10 +4,22 @@
  */
 
 import { ProjectService } from '@aws/workbench-core-accounts';
+import { isUserNotFoundError, UserManagementService } from '@aws/workbench-core-authentication';
+import { PermissionsService } from '@aws/workbench-core-authorization';
+import Boom from '@hapi/boom';
 import { Request, Response, Router } from 'express';
+import { validate } from 'jsonschema';
 import { wrapAsync } from './errorHandlers';
+import AssignUserToProject from './schemas/assignUserToProject';
+import { PermissionRoles } from './staticPermissionsConfig';
+import { processValidatorResult } from './validatorHelper';
 
-export function setUpProjectRoutes(router: Router, projectService: ProjectService): void {
+export function setUpProjectRoutes(
```

Comment:
nit: add tsdoc for better documentation

Reply:
What are we considering better in this scenario? The function and parameters seems like they're well named and descriptive.

Reply:
I wouldn't consider it better, just additional documentation

Reply:
What the additional documentation you're looking for here is what I mean haha.

Reply:
https://tsdoc.org/, description of setUpProjectRoutes, parameters, and return types, possible errors thrown

Reply:
ahh makes sense 👍

Reply:
updated